app/models/meeting.rb

```diff
@@ -22,9 +22,9 @@ class Meeting < ApplicationRecord
   validates :name, :description, :location, :time, :group_id, :date,
             presence: true
   belongs_to :group, foreign_key: :group_id
+  has_many :meeting_members, foreign_key: :meeting_id, dependent: :destroy
```

Comment:
This now has to be declared before the `has_many :members, through: :meeting_members` line below.